fastlane/Fastfile

```diff
@@ -409,60 +534,6 @@ platform :ios do
     end
   end
 
-  #####################################################################################
-  # build_and_upload_beta
-  # -----------------------------------------------------------------------------------
-  # This lane builds the app and uploads it for distribution
-  # -----------------------------------------------------------------------------------
-  # Usage:
-  # bundle exec fastlane build_and_upload_beta [skip_confirm:<skip confirm>]
-  #
-  # Example:
-  # bundle exec fastlane build_and_upload_beta
-  # bundle exec fastlane build_and_upload_beta skip_confirm:true
-  #####################################################################################
-  desc 'Builds and uploads for distribution'
-  lane :build_and_upload_beta do |options|
-    ios_build_prechecks(
-      skip_confirm: options[:skip_confirm],
-      internal: false,
-      external: true
-    )
-    ios_build_preflight
-    build_and_upload_to_app_store_connect(
-      skip_prechecks: true,
-      skip_confirm: options[:skip_confirm],
-      beta_release: true
-    )
-  end
-
-  #####################################################################################
-  # build_and_upload_release
-  # -----------------------------------------------------------------------------------
-  # This lane builds the app and uploads it for distribution
-  # -----------------------------------------------------------------------------------
-  # Usage:
-  # bundle exec fastlane build_and_upload_release [skip_confirm:<skip confirm>]
-  #
-  # Example:
-  # bundle exec fastlane build_and_upload_release
-  # bundle exec fastlane build_and_upload_release skip_confirm:true
-  #####################################################################################
-  desc 'Builds and uploads for distribution'
-  lane :build_and_upload_release do |options|
```

Comment:
After searching the release scenario, Buildkite scripts, and Buildkite pipelines, I did not find any instances of this lane being used.